CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 # Changelog
 
+# 1.0.0 (PROVISIONAL - STILL WORK IN PROGRESS)
+
+## Backwards Incompatible Changes
+
+### Configuration
+- **M3DB**: `db.bootstrap.bootstrappers` removed
+- **M3Coordinator**: `cluster.namespaces.storageMetricsType` removed
+- **M3Coordinator**: `tagOptions.tagOptions` no longer supports `legacy` type
+- **M3Query**: `limits.perQuery.maxComputedDatapoints` removed
+- **M3Query**: `limits.perQuery.maxFetchedDatapoints` removed
+- **M3Query**: `limits.global.maxFetchedDatapoints` removed
+- **M3Query**: `cache` removed
+- **M3Query**: `listenAddress` changed to always be resolved as a string from config. Format changed from
+```
+listenAddress:
+  config: "..."
+  value: "..."
+```
+to 
+```
+listenAddress: "..."
+```
+
+### API
+- **M3DB**: `/services/m3db/database/config/bootstrappers` dynamic bootstappers endpoint removed
+- **M3Coordinator**: `/api/v1/namespace` changed to `/api/v1/services/m3db/namespace`
+- **M3Coordinator**: `/api/v1/namespace/init` changed to `/api/v1/services/m3db/namespace/init`
+- **M3Coordinator**: `/api/v1/namespace/unagg` changed to `/api/v1/services/m3db/namespace/unagg`
+- **M3Coordinator**: `/api/v1/placement` changed to `/api/v1/services/m3db/placement`
+- **M3Coordinator**: `/api/v1/placement/init` changed to `/api/v1/services/m3db/placement/init`
```

Comment:
Can we be clear these were already deprecated and are now being removed?

Just say "Removed deprecated URL `foo` with stable preferred URL `bar`"?